tests/test_tlsfuzzer_expect.py

```diff
@@ -1538,9 +1632,12 @@ def test_process_with_tls_1_1_wrong_downgrade_protection(self):
 
         self.assertTrue(exp.is_match(msg))
 
-        with self.assertRaises(AssertionError):
+        with self.assertRaises(AssertionError) as e:
             exp.process(state, msg)
 
+        self.assertIn("downgrade protection sentinel but shouldn",
```

Comment:
incorrectly indented ?

Reply:
no? assertRaises does not abort execution, exactly so that raised exception can be inspected, as we do here